src/main/java/com/powsybl/openloadflow/dc/equations/ClosedBranchSide1DcFlowEquationTerm.java

```diff
@@ -58,7 +58,11 @@ public void update(double[] x) {
         double ph2 = x[ph2Var.getRow()];
         double a1 = a1Var != null ? x[a1Var.getRow()] : branch.getPiModel().getA1();
         p1 = calculate(ph1, ph2, a1);
-        rhs = -power * (A2 - a1);
+        if (a1Var == null) {
```

Comment:
In `ac` module, you have introduced the notion of active variable. I think that we have to test if `(a1Var != null && a1Var.isActive())`, as we do in `ac` module.

Reply:
Done